functions/utils/index.js

```diff
@@ -108,17 +46,19 @@ export const parseAssignedProperties = (properties) =>
     };
   }, {});
 
-export const fetchRecord = async (modelName, id, properties = []) => {
+export const fetchRecord = async (modelName, id, fragment = {}) => {
```

Comment:
Can we not solve this a better way instead of assigning an empty object if it does not exist? I would say it's better to just check if it's a valid value than the same for `${fragmentGql || ''}` ```${fragmentGql ? `...${name}` : ''}```